grass7/raster/r.fuzzy.set/r.fuzzy.set.html

```diff
@@ -117,31 +117,37 @@ <h4>Calculation of boundary shape</h4>
 <h2>SEE ALSO</h2>
 
 <em>
-<a href="r.fuzzy.logic.html">r.fuzzy.logic</a>,
+<a href="r.fuzzy.logic.html">r.fuzzy.logic</a> addon,
+<a href="r.fuzzy.system.html">r.fuzzy.system</a> addon,
 <a href="r.mapcalc.html">r.mapcalc</a>,
 </em>
 
 
 <h2>REFERENCES</h2>
 
-<p>Zadeh, L.A. (1965). "Fuzzy sets". Information and Control 8 (3): 338–353.
-doi:10.1016/S0019-9958(65)90241-X. ISSN 0019-9958.</P>
+<ul>
+<li>Jasiewicz, J. (2011). A new GRASS GIS fuzzy inference system for massive data analysis.
+Computers & Geosciences (37) 1525-1531. DOI <a href="http://10.1016/j.cageo.2010.09.008">http://10.1016/j.cageo.2010.09.008</a>
 
-<p>Novák, Vilém (1989). Fuzzy Sets and Their Applications. Bristol: Adam Hilger.
-ISBN 0-85274-583-4.</P>
+<li>Zadeh, L.A. (1965). "Fuzzy sets". Information and Control 8 (3): 338–353.
+doi:10.1016/S0019-9958(65)90241-X. ISSN 0019-9958.
 
-<p>Klir, George J.; Yuan, Bo (1995). Fuzzy sets and fuzzy logic: theory and
-applications. Upper Saddle River, NJ: Prentice Hall PTR. ISBN 0-13-101171-5.</P>
+<li>Novák, Vilém (1989). Fuzzy Sets and Their Applications. Bristol: Adam Hilger.
```

Comment:
AFAIK accented chars will fail: HTML codes are needed.

Reply:
Leaving for now - in my (limited) experience modern browsers handle UTF-8 fine.